clang/find_clang.cr

```diff
@@ -42,20 +42,17 @@ def print_help_and_bail
 end
 
 # Find clang++ binary, through user setting, or automatically.
-if binary = ENV["CLANG"]?
-  unless Process.find_executable(binary)
-    print_help_and_bail
-  end
-
-  clang_binary = binary
-elsif binary = find_clang_binary
-  clang_binary = binary
-else
-  print_help_and_bail
-end
+clang_binary = ENV["CLANG"]? || find_clang_binary
+print_help_and_bail unless clang_binary && Process.find_executable(clang_binary)
 
 STDERR.puts "Using clang binary #{clang_binary.inspect}"
 
+# Find llvm-config binary
+llvm_config = ENV["LLVM_CONFIG"]
```

Comment:
Missing `?`, this should behave like the "CLANG" check above and provide some help text to the user